editors/micro/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/zyedidia/micro 2.0.4 v
+
+homepage            https://micro-editor.github.io/
+
+description         A modern and intuitive terminal-based text editor
+
+long_description    micro is a terminal-based text editor that aims to be \
+                    easy to use and intuitive, while also taking advantage of \
+                    the capabilities of modern terminals. As its name \
+                    indicates, micro aims to be somewhat of a successor to \
+                    the nano editor by being easy to install and use. Micro \
+                    supports a full-blown plugin system. Plugins are written \
+                    in Lua and there is a plugin manager to automatically \
+                    download and install your plugins for you.
+
+categories          editors
+license             MIT
+
+build.target        github.com/go-delve/delve/cmd/dlv
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+build.cmd           make
+build.pre_args      {}
+build.args          build
+build.post_args     {}
+installs_libs       no
+use_parallel_build  no
```

Comment:
Why is this required ? If a bug in the build is there an upstream ticket ?

Reply:
I originally added this because I wanted to ensure make targets proceeded in proper sequence, but I've removed it.